src/components/buttons/primary.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+export interface PrimaryButtonProps {
+  label: string | React.ReactNode;
+  onClick: () => void;
+  disabled?: boolean;
+}
+
+const PrimaryButton: React.FC<PrimaryButtonProps> = ({
+  label,
+  onClick,
+  disabled,
+}) => {
+  return (
+    <button
+      className={`text-gdk-white bg-gdk-blue hover:bg-gdk-light-blue disabled:bg-gdk-light-gray 
+      pointer-events-auto my-4 flex h-[51px] w-fit items-center justify-center rounded-[10px] px-8 font-semibold`}
```

Comment:
i think we don't necessary need the pointer `events-auto-class` here (and in every button); I'd rather add it once to the parent container in which the buttons will be added to

Reply:
yes, sounds good!